Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -544,7 +549,11 @@ export class ResourceGroup {
                             return this.waitAndPerformAzureDeployment(armClient, deployment, retryCount);
                         }
                         this.writeDeploymentErrors(error);
-                        this.checkAndPrintPortalDeploymentURL(result.error);
+                        if(!!result && !!result.error) {
```

Comment:
is this an existing bug? do we need to hotfix it?

Reply:
It also needs to get ported to M163 and master

Reply:
No need to hotfix, as it doesn't break anything. M163 branch is not cut yet. I'll port to master